app.rb

```diff
@@ -0,0 +1,23 @@
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    erb :index
+  end
+
+  post '/sign_up' do
+    $username = params[:username]
+    $password = params[:password]
+    erb :sign_up
+  end
+
+  post '/peeps' do
+    Peep.create(peep: params[:peep])
+    @peeps = Peep.all.reverse
```

Comment:
is .reverse good enough to ensure reverse chronological order, what if peep database table got unsorted?